etl-driver/src/main/java/com/wipro/ats/bdre/im/etl/api/sftp/SFTP.java

```diff
@@ -129,11 +128,12 @@ private void transferFile(Session session, String remoteFile, String localFile)
                             // error
                             break;
                         }
-                        if (buf[0] == ' ') break;
+                        if (buf[0] == ' ')
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)